xcp_d/workflow/anatomical.py

```diff
@@ -269,1218 +297,1136 @@ def init_anatomical_wf(
-            # apply FNIRT-format warpfield
-            lh_native_apply_warpfield = pe.Node(
-                ApplyWarpfield(),
-                name="lh_native_apply_warpfield",
-                mem_gb=mem_gb,
-                n_procs=omp_nthreads,
-            )  # TM
-
-            rh_native_apply_warpfield = pe.Node(
-                ApplyWarpfield(),
-                name="rh_native_apply_warpfield",
-                mem_gb=mem_gb,
-                n_procs=omp_nthreads,
-            )  # TM
-
-            fs_std_mesh_L = str(
-                get_template(
-                    template="fsaverage",
-                    hemi="L",
-                    space=None,
-                    density="164k",
-                    desc=None,
-                    suffix="sphere",
-                )
-            )
+        # fmt:off
+        workflow.connect([
+            (collect_lh_surfaces, lh_native_apply_affine, [("out", "in_file")]),
+            (update_xform_wf, lh_native_apply_affine, [("outputnode.world_xform", "affine")]),
+        ])
+        # fmt:on
 
-            fs_std_mesh_R = str(
-                get_template(
-                    template="fsaverage",
-                    hemi="R",
-                    space=None,
-                    density="164k",
-                    desc=None,
-                    suffix="sphere",
-                )
-            )
+        rh_native_apply_affine = pe.MapNode(
+            ApplyAffine(),
+            name="rh_native_apply_affine",
+            mem_gb=mem_gb,
+            n_procs=omp_nthreads,
+            iterfield=["in_file"],
+        )  # TM
 
-            fs_L2fsLR = pkgrf(
-                "xcp_d",
-                (
-                    "data/standard_mesh_atlases/fs_L/"
-                    "fs_L-to-fs_LR_fsaverage.L_LR.spherical_std.164k_fs_L.surf.gii"
-                ),
-            )
+        # fmt:off
+        workflow.connect([
+            (collect_rh_surfaces, rh_native_apply_affine, [("out", "in_file")]),
+            (update_xform_wf, rh_native_apply_affine, [("outputnode.world_xform", "affine")]),
+        ])
+        # fmt:on
 
-            fs_R2fsLR = pkgrf(
-                "xcp_d",
-                (
-                    "data/standard_mesh_atlases/fs_R/"
-                    "fs_R-to-fs_LR_fsaverage.R_LR.spherical_std.164k_fs_R.surf.gii"
-                ),
-            )
+        # apply FNIRT-format warpfield
+        # NOTE: There are no connections going from these nodes back into the workflow.
+        lh_native_apply_warpfield = pe.MapNode(
+            ApplyWarpfield(),
+            name="lh_native_apply_warpfield",
+            mem_gb=mem_gb,
+            n_procs=omp_nthreads,
+            iterfield=["in_file"],
+        )
+        rh_native_apply_warpfield = pe.MapNode(
+            ApplyWarpfield(),
+            name="rh_native_apply_warpfield",
+            mem_gb=mem_gb,
+            n_procs=omp_nthreads,
+            iterfield=["in_file"],
+        )

```

Comment:
@madisoth do you know why these nodes do not have outgoing connections? Is this branch of the workflow unused?

Reply:
They're unused and can be removed. 

At one point we planned to output surfaces at 3 densities as in HCP (native, 164k, 32k) which this is left over from